docs/api/effector/combine.md

```diff
@@ -4,7 +4,7 @@ title: combine
 hide_title: true
 ---
 
-# `combine(...stores, fn)`
+# `combine(...stores, fn?)`
```

Comment:
It’s not correct: you can use either object/array with stores (one argument, and optional function) or multiple arguments in which each argument is store and the last is a function (not optional this time)
we have each real case covered https://github.com/zerobias/effector/blob/master/src/effector/__tests__/combine.test.js#L15

in short: this overload should not change, but all other changes are ok 👍

Reply:
Actually, this signature confuses me with `combine(...store, fn)`. I thought that fn was necessary, but as it turned out, if you cast it, you get an array.